kornia/geometry/camera/pinhole.py

```diff
@@ -153,7 +150,7 @@ def tx(self) -> Tensor:
         return self.extrinsics[..., 0, -1]
 
     @tx.setter
-    def tx(self, value: Union[Tensor, float, int]) -> 'PinholeCamera':
+    def tx(self, value: Union[Tensor, float]) -> 'PinholeCamera':
```

Comment:
are we dropping `int` here (and below) ? /cc @cjpurackal

Reply:
nops 
> In Python, int is a subtype of float, and float is a subtype of complex. As such, a union that includes both int and float is redundant, as it is equivalent to a union that only includes float.

from https://beta.ruff.rs/docs/rules/redundant-numeric-union/ - `PYI041`